src/index.html

```diff
@@ -17,11 +17,333 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
```

Comment:
You are linking a SCSS file in your HTML. Browsers cannot interpret SCSS, they only understand CSS. You should compile your SCSS to CSS and then link the CSS file in your HTML.